openslides/urls.py

```diff
@@ -12,21 +12,20 @@
     '',
     url(r'^rest/', include(router.urls)),
     url(r'^users/', include('openslides.users.urls')),
-
     url(r'^users.*', IndexView.as_view()),
+    url(r'^assignments/', include('openslides.assignments.urls')),
+    url(r'^assignments.*', IndexView.as_view()),
+    url(r'^agenda/', include('openslides.agenda.urls')),
+    url(r'^agenda.*', IndexView.as_view()),
```

Comment:
Please add a TODO that all patterns with end with ".*" can be removed after we add the global url-pattern "/" to the indexView